script/output/holesky/M2_deploy_from_scratch.output.json

```diff
@@ -45,7 +45,16 @@
     "strategyManager": "0xdfB5f6CE42aAA7830E94ECFCcAd411beF4d4D5b6",
     "strategyManagerImplementation": "0x59f766A603C53f3AC8Be43bBe158c1519b193a18",
     "rewardsCoordinator": "0x0000000000000000000000000000000000000000",
-    "rewardsCoordinatorImplementation": "0x0000000000000000000000000000000000000000"
+    "rewardsCoordinatorImplementation": "0x0000000000000000000000000000000000000000",
```

Comment:
?

Reply:
are you question marking the diff here? the whole line is marked as a diff because a comma was added at the end :D